processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2025-04-14 18:51:28.987533301 -0400
+++ memcpu.hex	2025-04-14 18:51:28.971532690 -0400
@@ -19,7 +19,7 @@
 :0400860000732023C0
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
+:040101000000BEEF4D
 :040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/msi1.asm (1 of 1)
processing file asmFiles/msi1.asm (1 of 2)
processing file asmFiles/msi2.asm (2 of 2)
processing file asmFiles/dual.mergesort.asm (1 of 1)
processing file asmFiles/dual.mergesort.asm (1 of 1)
--- memsim.hex	2025-04-14 18:53:12.235478656 -0400
+++ memcpu.hex	2025-04-14 18:53:12.214477853 -0400
@@ -176,64 +176,64 @@
 :040181000000000377
 :040182000000000475
 :040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
+:040184000000000B6C
+:040185000000000C6A
+:040186000000001560
+:04018700000000165E
+:04018800000000165D
+:040189000000001959
+:04018A000000001B56
+:04018B000000001D53
+:04018C00000000204F
+:04018D00000000214D
+:04018E000000002944
+:04018F000000003339
+:040190000000000764
+:040191000000003832
+:040192000000000762
+:040193000000000761
+:04019400000000085F
+:040195000000000C5A
+:04019600000000085D
+:04019700000000164E
+:040198000000000A59
+:040199000000001949
+:04019A000000001051
+:04019B000000001D43
+:04019C000000001748
+:04019D00000000213D
+:04019E00000000203D
+:04019F00000000203C
+:0401A0000000002338
+:0401A1000000000753
+:0401A2000000002633
+:0401A3000000002632
+:0401A400000000282F
+:0401A5000000002F27
+:0401A6000000003025
+:0401A7000000003222
 :0401A800000000381B
 :0401A9000000003D15
 :0401AA000000003D14
 :0401AB000000003E12
 :0401AC00000000410E
-:0401AD00000000420C
+:0401AD000000001D31
 :0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
+:0401AF000000003D0F
+:0401B0000000004308
+:0401B1000000002129
 :0401B2000000004BFE
-:0401B3000000004DFB
+:0401B3000000003E0A
 :0401B40000000050F7
-:0401B50000000051F5
+:0401B5000000001D29
 :0401B60000000051F4
-:0401B70000000053F1
+:0401B7000000002024
 :0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
+:0401B9000000003D05
+:0401BA000000005AE7
+:0401BB000000000739
 :0401BC000000005CE3
-:0401BD0000000063DB
+:0401BD00000000211D
 :0401BE0000000064D9
 :0401BF0000000064D8
 :043FFE0000000600B9
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
--- memsim.hex	2025-04-14 18:54:19.225039175 -0400
+++ memcpu.hex	2025-04-14 18:54:19.199038181 -0400
@@ -15,10 +15,10 @@
 :04000E0018B2A42F51
 :04000F0000B52423F1
 :04001000FFFFFFFFF0
-:0400200000000002DA
-:0400220000000001D9
-:0400240000000002D6
-:0400260000000001D5
+:0400200000000001DB
+:0400220000000003D7
+:0400240000000001D7
+:0400260000000003D3
 :040080000000223723
 :04008100FC02021368
 :0400820009006513F9
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2025-04-14 18:54:21.504126287 -0400
+++ memcpu.hex	2025-04-14 18:54:21.486125599 -0400
@@ -176,64 +176,64 @@
 :040181000000000377
 :040182000000000475
 :040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
+:040184000000000B6C
+:040185000000000C6A
+:040186000000001560
+:04018700000000165E
+:04018800000000165D
+:040189000000001959
+:04018A000000001B56
+:04018B000000001D53
+:04018C00000000204F
+:04018D00000000214D
+:04018E000000002944
+:04018F000000003339
+:040190000000000764
+:040191000000003832
+:040192000000000762
+:040193000000000761
+:04019400000000085F
+:040195000000000C5A
+:04019600000000085D
+:04019700000000164E
+:040198000000000A59
+:040199000000001949
+:04019A000000001051
+:04019B000000001D43
+:04019C000000001748
+:04019D00000000213D
+:04019E00000000203D
+:04019F00000000203C
+:0401A0000000002338
+:0401A1000000000753
+:0401A2000000002633
+:0401A3000000002632
+:0401A400000000282F
+:0401A5000000002F27
+:0401A6000000003025
+:0401A7000000003222
 :0401A800000000381B
 :0401A9000000003D15
 :0401AA000000003D14
 :0401AB000000003E12
 :0401AC00000000410E
-:0401AD00000000420C
+:0401AD000000001D31
 :0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
+:0401AF000000003D0F
+:0401B0000000004308
+:0401B1000000002129
 :0401B2000000004BFE
-:0401B3000000004DFB
+:0401B3000000003E0A
 :0401B40000000050F7
-:0401B50000000051F5
+:0401B5000000001D29
 :0401B60000000051F4
-:0401B70000000053F1
+:0401B7000000002024
 :0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
+:0401B9000000003D05
+:0401BA000000005AE7
+:0401BB000000000739
 :0401BC000000005CE3
-:0401BD0000000063DB
+:0401BD00000000211D
 :0401BE0000000064D9
 :0401BF0000000064D8
 :043FFE0000000600B9
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/mergesort.asm (1 of 1)
